.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for your interest in the project. I appreciate bugs filed and PRs submitted!
+Please fill out this template with all the relevant information so we can
+understand what's going on and fix the issue.
+
+If you've never done that before, that's great! Check this https://www.opencue.io/contributing/opencue/contributing/
```

Comment:
I think we could delete this entire line. I don't believe it's necessary to review the contributing guidelines just to file a bug.